1.10/cli/command-reference/dcos-auth/dcos-auth-login.md

```diff
@@ -4,7 +4,7 @@ menu_order: 2
 ---
 
 # Description
-Log in to DC/OS authentication. 
```

Comment:
Do you think "log in to DC/OS authentication" makes sense? I'm thinking "log in to DC/OS" or "Authenticate to DC/OS" might be more precise.